src/Resources/ThreadsRuns.php

```diff
@@ -32,6 +36,25 @@ public function create(string $threadId, array $parameters): ThreadRunResponse
         return ThreadRunResponse::from($response->data(), $response->meta());
     }
 
+    /**
+     * Creates a streamed run
+     *
+     * @see https://platform.openai.com/docs/api-reference/runs/createRun
+     *
+     * @param  array<string, mixed>  $parameters
+     * @return EventStreamResponse<mixed>
+     */
+    public function createStreamed(string $threadId, array $parameters): EventStreamResponse
```

Comment:
This method can also be implemented into the `ThreadsRunsContract`

Reply:
Good idea, 
I did forget to add the functions into the contract. 
I'll go over the contracts when I add in support for the create and run endpoint.